src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/Message.vue

```diff
@@ -13,20 +13,35 @@
         </v-toolbar>
         <v-card-text>
           <h2>{{ messageStore.currentMessage?.subject }}</h2>
-          <p class="small mt-2">{{ messageDate }}</p>
-          <p class="small mt-6">
-            <span v-html="messageStore.currentMessage?.text"></span>
-          </p>
+
+          <div v-for="(message, index) in messageStore.currentThread" :key="index" class="small mt-6">
+            <span v-html="message.from == 'Registry' ? 'From ECE Registry' : 'PortalUser' ? 'You Replied' : ''"></span>
+            <br />
+            <br />
+            <span v-html="formatDate(String(message.notifiedOn), 'LLL d, yyyy t')"></span>
+            <br />
+            <br />
+            <br />
+            <span v-html="message.text"></span>
```

Comment:
**NITPICK** `<span v-html="message.text" class="mt-{margin}></span>` maybe cleaner? Same as `<span v-html="formatDate(String(message.notifiedOn), 'LLL d, yyyy t')"></span>`

Reply:
Agreed...I had already tried class="mt-{margin}" earlier with <span> tag and it did not work so resorted to using <br/>...changed <span> to <div> now to make it work..